components/popover/popover-mixin.js

```diff
@@ -13,21 +13,9 @@ export const PopoverMixin = superclass => class extends superclass {
 
 	static get properties() {
 		return {
-			/**
-			 * Whether the popover is open or not
-			 * @type {boolean}
-			 */
-			opened: { type: Boolean, reflect: true },
-			/**
-			 * Whether to disable auto-close/light-dismiss
-			 * @type {boolean}
-			 */
-			noAutoClose: { type: Boolean, reflect: true, attribute: 'no-auto-close' },
-			/**
-			 * Whether to disable auto-focus on the first focusable element when opened
-			 * @type {boolean}
-			 */
-			noAutoFocus: { type: Boolean, reflect: true, attribute: 'no-auto-focus' },
+			_noAutoClose: { state: true },
```

Comment:
If we want, we could name these internal state properties to be more popover specific.